cartography/intel/gcp/__init__.py

```diff
@@ -1,11 +1,35 @@
 from oauth2client.client import GoogleCredentials, ApplicationDefaultCredentialsError
+import googleapiclient.discovery
 import logging
 
-from cartography.intel.gcp import crm
+from cartography.intel.gcp import crm, compute
 
 logger = logging.getLogger(__name__)
 
 
+def _get_crm_resource_v1(credentials):
+    # cache_discovery=False to suppress extra warnings.
+    # See https://github.com/googleapis/google-api-python-client/issues/299#issuecomment-268915510 and related issues
+    return googleapiclient.discovery.build('cloudresourcemanager', 'v1', credentials=credentials, cache_discovery=False)
+
+
+def _get_crm_resource_v2(credentials):
+    """
+    v2 CRM client is required for querying folders
+    """
+    return googleapiclient.discovery.build('cloudresourcemanager', 'v2', credentials=credentials, cache_discovery=False)
+
+
+def _get_compute_resource(credentials):
+    return googleapiclient.discovery.build('compute', 'v1', credentials=credentials, cache_discovery=False)
+
+
+def _initialize_resources(credentials):
+    return {'crm_v1': _get_crm_resource_v1(credentials),
+            'crm_v2': _get_crm_resource_v2(credentials),
+            'compute': _get_compute_resource(credentials)}
```

Comment:
Not a fan of this pattern -- strings and mutable data structures are two places where bugs tend to sneak in. If you think it's important to pass around a group of clients like this then I would suggest creating a class (maybe a namedtuple or something similar). That way you get to leverage linting and are more likely to catch issues with test values.